db/migrate/20240306082827_create_transaction_categories.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTransactionCategories < ActiveRecord::Migration[7.2]
+  def change
+    create_table :transaction_categories, id: :uuid do |t|
+      t.string "name", null: false
+      t.string "icon", null: false
+      t.string "color", null: false
```

Comment:
Might be useful to default this to something like `#6172F3`?

(I didn't use gray because that will be used for "Uncategorized")